tests/openvino/test_modeling.py

```diff
@@ -1934,6 +1934,39 @@ def test_compare_to_transformers(self, model_arch):
             f"generation config : {gen_config}, transformers output {transformers_outputs}, ov_model output {ov_outputs}",
         )
 
+        # previous run was with legacy processing, one more run with features concatenation on preprocessing level
+        if (
+            model_arch in ["llava", "llava-next"]
+            and is_transformers_version(">=", "4.45")
+            and (processor.patch_size is None or processor.vision_feature_select_strategy is None)
+        ):
+            processor.patch_size = ov_model.config.vision_config.patch_size
+            processor.vision_feature_select_strategy = ov_model.config.vision_feature_select_strategy
+            if model_arch == "llava":
+                # testing model for llava does ot have specified image_seq_length and it is different from default
```

Comment:
This sentence does not seem right

Reply:
it means that testing model broken and does not contain required information. Model uses reduced shapes comparing with original llava, but default config does not follow recalculation based on model parameters for this parameter and hardcode value from original model.

I think better to replace model with fix

Reply:
test model confg updated to store correct value